### PR TITLE
Spucelik/power apps

### PR DESCRIPTION
**Pull request by Spucelik**
Initial upload of Invoice Automation solution

### CODE REVIEW

File: PowerApps/Invoice Automation/Azure-FunctionAppDeployment.ps1 [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#http://www.redbaronofazure.com/?cat=428
+#https://demiliani.com/2020/03/26/a-quick-way-to-deploy-your-azure-functions-in-the-cloud/
+
+
+$location = "East US"
+$resourceGroupName = "InvoiceAutomation"
+$storageAccountName = "invoicenautomationrg"
+$functionName = "InvoiceLineItems"
+$sourceZipPath = "C:\Users\stpuceli\source\repos\HelloWorld\bin\Release\netcoreapp3.0\publish\InvoiceLineItems.zip"
```
Comment (shanepeckham) [inline PowerApps/Invoice Automation/Azure-FunctionAppDeployment.ps1:9]: This is a path on your machine, can you add instructions to unzip the relevant file to a path and then set here please

File: PowerApps/Invoice Automation/readme.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+<h1>Invoice Automation using the Power Platform</h1>
+
+Invoice processing is something that every company does, but many are still doing it manually and is very labor intensive.  Not anymore!  This starter kit is intended to use the new "Process and save information from invoices" flow in Power Automate to automatically capture invoice information, save it to a SharePoint document library.  We’ll then use Cognitive Services to extract the invoice line-item information and save it to a SharePoint List.  Finally, Power Apps will allow you to review and complete the processing so it can be integrated with the accounting LOB application.
+
+For an overview of the solution please refer to this [blog article]("https://powerusers.microsoft.com/t5/Power-Automate-Community-Blog/Invoice-Automation-using-the-Power-Platform/ba-p/875628) for details.
```
Comment (shanepeckham) [inline PowerApps/Invoice Automation/readme.md:5]: Any chance you can add the overview here as well? You can keep the blog link in but this repo is intended to be the single source of all info, so the user should be able to find all info in this README

File: PowerApps/Invoice Automation/readme.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+Deploy the Azure Function
+    <ol>
+        <li>Place the InvoiceItems.zip file in a local folder and reference it in the PathToZipFile $sourceZipPath variable</li>
+        <li>The azure function can be deployed using the below PowerShell Script.
+
+        $location = "East US"
+		$resourceGroupName = "InvoiceAutomation"
+		$storageAccountName = "invoicenautomationrg"
+		$functionName = "InvoiceLineItems"
+		$sourceZipPath = "<PathToZipFile>\InvoiceLineItems.zip"
```
Comment (shanepeckham) [inline PowerApps/Invoice Automation/readme.md:56]: Apply the same $sourceZipPath you have here to the file above

File: PowerApps/readme.md [hunk 4]
```diff
@@ -0,0 +1 @@
+test
```
Comment (shanepeckham) [inline PowerApps/readme.md:1]: Can you add the ToC here please

File: README.md [hunk 5]
```diff
@@ -267,6 +267,7 @@ used in the code. Introduction and/or reference of those will be provided in the
     <td align="center"><a href="https://github.com/mover333"><img src="https://avatars.githubusercontent.com/u/55155635?s=460&v=4" width="100px;" alt=""/><br /><sub><b>Mitchell Overfield</b></sub></a><br /></td>
     <td align="center"><a href="https://github.com/alkofahi"><img src="https://avatars.githubusercontent.com/u/58956796?s=460&v=4" width="100px;" alt=""/><br /><sub><b>Jafar Al-Kofahi</b></sub></a><br /></td>
```
Comment (shanepeckham) [inline README.md:268]: You also need to add a new section to this root README. Look at the End to end demos ✨
NEW (▀̿Ĺ̯▀̿ ̿) section, basically copy that and call it PowerApps and feel free to add a new icon to it :)

**Review by shanepeckham — CHANGES_REQUESTED** [review]

File: PowerApps/Invoice Automation/readme.md (hunk 2 above)
Comment (Spucelik) [reply PowerApps/Invoice Automation/readme.md:7]: Yes...I'll try to have that in this week.

File: README.md (hunk 5 above)
Comment (Spucelik) [reply README.md]: I'll add it tonight.

**Pull request merged**